etc/DockerHelper.sh

```diff
@@ -56,9 +56,12 @@ _setup() {
         "ubuntu20")
             osBaseImage="ubuntu:20.04"
             ;;
-        "ubuntu22")
+        "ubuntu22.04")
```

Comment:
Let's use the same convention for all ubuntu versions and have the minor release as well on lie 56 and where else it is used.